src/components/Form.js

```diff
@@ -88,14 +88,15 @@ export default class Form extends Component {
   }
 
   validate(formData, schema = this.props.schema) {
-    const { validate, transformErrors } = this.props;
+    const { validate, transformErrors, localization } = this.props;
     const { definitions } = this.getRegistry();
     const resolvedSchema = retrieveSchema(schema, definitions, formData);
     return validateFormData(
       formData,
       resolvedSchema,
       validate,
-      transformErrors
+      transformErrors,
+      localization
```

Comment:
Why are you returning `localization` from `validateFormData` (i.e. what is the use case for doing so)?